UMN-Grouper.psm1

```diff
@@ -376,6 +376,95 @@ function Get-GrouperPrivileges
     }
 #endregion
 
+#region Get-GrouperStemByParent
+    function Get-GrouperStemByParent
+    {
+        <#
+            .SYNOPSIS
+                Get Grouper chled Stem(s) of a parent stem
+
+            .DESCRIPTION
+                Get Grouper chled Stem(s) of a parent stem
```

Comment:
```suggestion
                Get Grouper child Stem(s) of a parent stem
```

Reply:
I'll test the case of a single a single return value.  Do you have a specific value you used so I an duplicate your test as?